### PR TITLE
Audits `XrpPaymentChannelCreate` Protobuf Wrapper

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR adds additional logic validations and descriptive error throwing to the class `XrpPaymentChannelCreate`.

### Context of Change

Another component of our attempt to make protobuf wrapper classes as safe and informative as possible. An SDK safety net.

### Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)

## Before / After

`XrpPaymentChannelCreate` performs logic/error checking, based on the business logic in the docs.

## Test Plan

New tests added to confirm that errors are correctly thrown. Existing tests were edited to properly account for the new errors. CI passes.

## Future Tasks
This is part of a series, with all subtasks listed here: https://app.asana.com/0/1188384817180008/1193021408531788/f

### CODE REVIEW

**Review by tedkalaw — APPROVED** [review]
do we need to handle the ambiguous falsy cases here as well?

File: src/XRP/model/xrp-payment-channel-create.ts [hunk 1]
```diff
@@ -42,13 +49,26 @@ export default class XrpPaymentChannelCreate {
...
+        XrpErrorType.MalformedProtobuf,
+        'Cannot construct XAddress from PaymentChannelCreate protobuf `destination` and `destinationTag` fields.',
+      )
+    }
 
     const settleDelay = paymentChannelCreate.getSettleDelay()?.getValue()
+    if (settleDelay === undefined) {
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'PaymentChannelCreate protobuf does not contain `SettleDelay` field.',
```
Comment (amiecorso) [inline src/XRP/model/xrp-payment-channel-create.ts:63]: lowercase SettleDelay maybe?  you can see that these fields are all always lower case https://github.com/ripple/rippled/blob/3d86b49dae8173344b39deb75e53170a9b6c5284/src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L278

File: test/XRP/xrp-transaction-type-proto-conversion.test.ts [hunk 2]
```diff
@@ -1006,16 +1010,59 @@ describe('Protobuf Conversions - Transaction Types', function (): void {
...
+    assert.throws(() => {
+      XrpPaymentChannelCreate.from(
+        testInvalidPaymentChannelCreateProtoBadDestination,
+        XrplNetwork.Test,
+      )
+    }, XrpError)
+  })
+
+  it('Convert PaymentChannelCreate protobuf to XrpPaymentChannelCreate object - missing PublicKey', function (): void {
+    // GIVEN a PaymentChannelCreate protocol buffer missing the PublicKey field.
```
Comment (amiecorso) [inline test/XRP/xrp-transaction-type-proto-conversion.test.ts:1047]: obvs an insignificant test comment, but just noticing the field casing here too and same for PublicKey

**Review by amiecorso — APPROVED** [review]
looks great just some of my same super nits about casing

**Pull request merged**